src/assets/wise5/authoringTool/peer-grouping/edit-peer-grouping-dialog/edit-peer-grouping-dialog.component.ts

```diff
@@ -25,14 +31,63 @@ export class EditPeerGroupingDialogComponent extends AuthorPeerGroupingDialogCom
   ngOnInit(): void {
     this.peerGrouping = this.utilService.makeCopyOfJSONObject(this.peerGrouping);
     this.stepsUsedIn = this.peerGroupingAuthoringService.getStepsUsedIn(this.peerGrouping.tag);
+    this.logicType = this.getLogicType(this.peerGrouping.logic);
+    if (this.logicType === DIFFERENT_IDEAS_VALUE) {
```

Comment:
The variable name DIFFERENT_IDEAS_VALUE is a bit confusing to me here, because what is it a value of? Can it just be "differentIdeas" (string, instead of using variable)? Or how about renaming the variable to DIFFERENT_IDEAS?